templates/conventions/actions/validate_resiliation.html

```diff
@@ -1,5 +1,5 @@
 {% load custom_filters %}
-{% if convention|display_validation:request %}
+{% if True %}
```

Comment:
On peut remove ce if non ? Ou alors t'as changé la condition pour le développement ?

Reply:
```suggestion
{% if convention|display_validation:request %}
```

Merci pour la review :)